src/components/CheckBox/Checkbox.module.scss

```
@@ -0,0 +1,46 @@
+@import '../../styles/variables.scss';
```

[+] Попробуй так:
```suggestion
@import 'src/styles/variables.scss;
```
Аналогично с другими файлами со стилями